webapp/angular.json

```diff
@@ -40,7 +43,6 @@
               "optimization": true,
               "outputHashing": "all",
               "sourceMap": false,
-              "extractCss": true,
```

Comment:
@rahul-rathore-576 why we removed that one?

Reply:
@evereq Angular version (11.2.13) `ng update` CLI automatically removed `extractCss` option from angular.json by itself.